resources/migrations/000_migrations.yaml

```diff
@@ -11308,6 +11308,26 @@ databaseChangeLog:
                   name: remark
                   type: ${text.type}
                   remarks: 'Optional remarks explaining why these changes were made.'
+  - changeSet:
+      id: v43.00-047
+      author: qnkhuat
+      comment: Added 0.43.0. Grant permissions to create/edit subscription to all existing groups except Admin
```

Comment:
Per my top-level comment, I think this migration only needs to add subscription permissions for the `All Users` group.